app/assets/javascripts/discourse/controllers/preferences/users.js.es6

```diff
@@ -6,6 +6,12 @@ import User from "discourse/models/user";
 export default Ember.Controller.extend(PreferencesTabController, {
   saveAttrNames: ["muted_usernames", "ignored_usernames"],
   ignoredUsernames: Ember.computed.alias("model.ignored_usernames"),
+  userIsMemberOrAbove: Ember.computed.gte("model.trust_level", 2),
+  userIsStaffOrModerator: Ember.computed.or("model.staff", "model.moderator"),
```

Comment:
Hmm, usually the condition is `staff = moderator || admin`. So checking only for staff should be ok?

Reply:
Ah! Right 👍